pages/samvera/developer_community/release_process.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Hyrax Testing and Release Process
+a-z: [Testing, Release Process, Managers, Product Owners]
+last_updated: February 5, 2018
+tags: [release, community]
+sidebar: samvera_sidebar
+permalink: release_process.html
+folder: samvera
+toc: false
+---
+<br />
+
+The [Repository Managers Interest Group](https://wiki.duraspace.org/display/samvera/Repository+Management+Interest+Group) will work with the Hyrax PO to test and document features for Hyrax releases. For each release cycle, there will be at least one designated person from the Repo-Managers to facilitate the process. [Instructions for conducting release testing can be found here](/release_testing.html) (in progress).
```

Comment:
I would drop `(in progress)` here and put it on the release_testing page itself.  It would be easy to forget to come back later and take out the in progress comment.